### PR TITLE
feat(release): Remove poetry publish

### PR DESCRIPTION
**Pull request by esloch**

### CODE REVIEW

File: .releaserc.json [hunk 1]
```diff
@@ -49,14 +49,7 @@
...
         {
-          "prepareCmd": "poetry build",
-          "publishCmd": "poetry publish"
-        }
-      ],
-      [
-        "@semantic-release/github",
-        {
-          "assets": ["dist/*.whl", "dist/*.tar.gz"]
+          "prepareCmd": "poetry build"
```
Comment (xmnlab) [inline .releaserc.json:52]: probably you don't need to use poetry build here.
```
    [
      "@semantic-release/exec",
      {
        "prepareCmd": "poetry build"
      }
    ],
```

->
```
    "@semantic-release/exec",
```

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -11,47 +8,51 @@ on:
...
-    - uses: conda-incubator/setup-miniconda@v2
-      with:
-        miniconda-version: "latest"
-        mamba-version: "*"
-        environment-file: conda/dev.yaml
-        channels: conda-forge,nodefaults
-        activate-environment: alertadengue
-        use-mamba: true
-        miniforge-variant: Mambaforge
+      - uses: conda-incubator/setup-miniconda@v2
```
Comment (xmnlab) [inline .github/workflows/release.yaml:30]: indentation here looks wrong

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -11,47 +8,51 @@ on:
...
-        npx --yes \
-          -p semantic-release \
-          -p "@semantic-release/commit-analyzer" \
-          -p "@semantic-release/release-notes-generator" \
-          -p "@semantic-release/changelog" \
-          -p "@semantic-release/exec" \
-          -p "@semantic-release/github" \
-          -p "@semantic-release/git" \
-          -p "@google/semantic-release-r
+      - name: Release
```
Comment (xmnlab) [inline .github/workflows/release.yaml:44]: indentation here looks wrong

**Comment by esloch** [comment]
Thanks for you review @xmnlab!

**Pull request merged**